.buildkite/premerge.steps.yaml

```diff
@@ -73,6 +73,12 @@ steps:
             type: details
     <<: *linux
 
+  - label: "build deployment launcher :rocket:"
```

Comment:
you know what to do

![](https://i.giphy.com/media/26BRBAA6bQv8nqA4o/giphy.webp)

Reply:
I'm gonna pretend like I didn't see this.